packages/@okta/vuepress-site/.vuepress/nav/main.js

```diff
@@ -44,6 +44,22 @@ module.exports = [
       },
       { title: 'Manage Okta Resources', link: '/docs/reference/api/roles/',  subLinksId: 'Sidebar_Resources',  subLinks: [
           { title: 'Administrator Roles', link: '/docs/reference/api/roles/'},
+          { title: 'Advanced Server Access', link: '/docs/reference/api/asa/', subLinksId: 'Sidebar_Resources', subLinks: [
+            { title: 'Attributes', link: '/docs/reference/api/asa/attributes/'},
+            { title: 'Groups', link: '/docs/reference/api/asa/groups/'},
+            { title: 'Projects', link: '/docs/reference/api/asa/projects', subLinksId: 'Sidebar_Resources', subLinks: [
+                { title: 'Client Configuration Options', link: '/docs/reference/api/asa/projects/client_config_options/'},
+                { title: 'Cloud Accounts', link: '/docs/reference/api/asa/projects/cloud_accounts/'},
```

Comment:
Our URLs should use hyphens, not underscores. `cloud_accounts` -> `cloud-accounts`